site/pages/api/hootsuite-callback.api.ts

```diff
@@ -101,10 +99,13 @@ const hootsuiteCallback = async (req: NextApiRequest, res: NextApiResponse) => {
 
         const userId: string = userDetails.data.id;
 
-        const key = `/social/${session.orgId}/hootsuite/${userId}-${
+        const key = `/social/${session.orgId}/hootsuite/${userId}-token`;
+
+        const addedByKey = `/social/${session.orgId}/hootsuite/${userId}-addedUser-${
```

Comment:
Do we need the prefix after addedUser? If another user logs in then are we invoking the `deleteParameter` api for this user?

Reply:
yes for clarity is someone checks ssm to understand what that value is and it will get overwritten when another user connects

Reply:
As parameter value will contain the user name - this could be used to check the value.  

For overwriting, we are calling deleteParameter and then putParameter - If this is done to overcome eventual consistency in SSM then its fine but if we are not facing the issue while updating the existing parameter then we can avoid the redundant deleteParamter api call

Reply:
done